Sources/swift-bundler/Commands/BundleArguments.swift

```diff
@@ -73,6 +73,12 @@ struct BundleArguments: ParsableArguments {
       return "The platform to build for \(possibleValues). (default: macOS)"
     }(),
     transform: { string in
+      // also support getting a platform by its apple sdk equivalent.
+      if let appleSDK = AppleSDKPlatform(rawValue: string)
+      {
```

Comment:
nuh uh

Reply:
oh my gosh I didn't even notice it 😆 I'm scared to run it, is your repo set up to take a `swiftformat .`, because I'm tempted to forcefully add that as apart of a new `git` commit command alias function.